lib/entry/StateManager.js

```diff
@@ -77,14 +79,17 @@ module.exports = function(entryObject, playlistGenerator) {
     }
 
     function onplay(event, from, to) {
-        if (from ==="suspending"){
+        if (from === "suspending"){
             logger.warn("Something went wrong:" + stateMachineMessagePattern, event.toUpperCase(), from, to)
         }
         var currTime = new Date().getTime();
         if ((lastServerUpdate === null) || ((currTime - lastServerUpdate) >= serverUpdateInterval)) {
-            // When reporting play update entry's playlist
-            playlistGenerator.updatePlaylist();
             logger.info(stateMachineMessagePattern, event.toUpperCase(), from, to);
+            // When reporting play update entry's playlist very 5 min
+            if (shouldUpdatePlaylist % config.get('playlistUpdateTimeInMin') === 0) {
```

Comment:
playlistUpdateTimeInMin?  but this is multiply by serverUpdateInterval... so if serverUpdateInterval is 12000 then it's two minutes